components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -124,13 +128,21 @@ const confirm = ref(false)
 const days = [7, 14, 30]
 const selectedDay = ref(14)
 
+const insufficientBalance = computed(
+  () => Number(offerPrice.value) > simpleDivision(balance.value, decimals.value)
+)
+
+const offerPriceInvalid = computed(() => {
+  if (offerPrice.value) {
+    return Math.sign(offerPrice.value) !== 1
+  }
+  return true
```

Comment:
Is it different from `const offerPriceInvalid = computed(() => offerPrice.value && offerPrice.value>0)`
I guess it would be easier to read.

Reply:
yes it's different, the result of your suggested change returns undefined if there's no price (the beginning state) and the button is therefore enabled

Reply:
Oh, I got it. 👍